packages/api-client/package.json

```diff
@@ -107,6 +107,8 @@
   ],
   "module": "dist/index.js",
   "dependencies": {
+    "@codemirror/state": "^6.4.0",
+    "@codemirror/view": "^6.23.1",
```

Comment:
these are already added within `use-codemirror` component, should we maintain that change?

Reply:
im not entirely sure tbh

Reply:
probs not but we would need to export them : )